cameras:
  - name: cs050
    serial_number: "DA6320570"
    exposure:
      auto: true
      min: 15
      max: 13319
      mode: 0
      value: 5000.0
    gain:
      auto: true
      min: 0.0
      max: 22.0
      value: 6.0
    white_balance:
      auto: true
    roi:
      width: 2448
      height: 2048
      offset_x: 0
      offset_y: 0

loop_rate_hz: 60